packages/odyssey-react/src/components/Tooltip/Tooltip.module.scss

```
@@ -11,22 +11,23 @@
  */
 
 .root {
-  @include border-radius($base-border-radius);
+  @include border-radius(var(--TipBorderRadius));
 
   display: block;
   visibility: hidden;
   position: absolute;
   z-index: 1;
-  padding-block: $spacing-xs;
-  padding-inline: $spacing-s;
-  transition: opacity $base-duration $base-timing 1s;
-  border: 1px solid cv("gray", "900");
+  padding-block: var(--TipPaddingBlock);
+  padding-inline: var(--TipPaddingInline);
+  transition: opacity var(--TransitionDuration) var(--TransitionTiming)
+    var(--TransitionDelay);
+  border: var(--TipBorderWidth) solid var(--TipBackgroundColor);
   opacity: 0;
-  background: cv("gray", "900");
-  color: $text-body-inverse;
-  font-size: $size-body-caption;
-  font-weight: 600;
-  line-height: $title-line-height;
+  background: var(--TipBackgroundColor);
```

can we change this to avoid the shorthand declaration? I'm just not sure we want to open up that level of flexibility here:

```suggestion
  background-color: var(--TipBackgroundColor);
```